scylla/src/transport/connection_pool.rs

```diff
@@ -97,6 +100,8 @@ impl NodeConnectionPool {
         let (use_keyspace_request_sender, use_keyspace_request_receiver) = mpsc::channel(1);
         let pool_updated_notify = Arc::new(Notify::new());
 
+        let keepalive_interval = pool_config.keepalive_interval;
```

Comment:
nitpick: I wonder if we should generate a warning when somebody tries to set a keepalive interval that we subjectively judge as ridiculous - e.g. 1ms or 1s. I think that anything <= 1s is worthy of a warning, because it would produce a non-zero amount of noise in metrics and unnecessary throughput.

Reply:
(totally optional, also fine as a follow-up, although it's also a super-simple change if you decide that it makes sense)